config/environments/production.rb

```diff
@@ -29,7 +29,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
If `config.assets.compile` needs to be set to `true` in production then there's an underlying issue that needs to be addressed.

For performance reasons, the assets ought to be compiled only *once* for production. The asset compile step happens during the deploy phase (i.e. during `git push heroku master`). If `config.assets.compile` is set to true, the compile step is happening more than once and this can lead to performance issues in production.

I recommend setting `config.assets.compile` back to false and determine what the underlying issue is and addressing that.